woocommerce/woocommerce-gutenberg-products-block
# Convert Title component to TypeScript

#### Description

Converting the `<Title />` component from JS to TypeScript.

### Testing

#### Automated Tests

### Manual Testing

Please regression test this component to ensure it is still working as expected. You can also do this by comparing the behaviour to that on the `trunk` branch.

1. Checkout the branch and check that the compiler reports no TS errors associated with the converted files.
2. Add block(s) to your page which uses the component. Such as the `All Products` block for example (the more variety the better)
2. Test no regressions with the title components behaviour have been introduced.

### Changelog

> Title: Convert to TypeScript
